packages/ember-glimmer/lib/helpers/-class.js

```diff
@@ -3,14 +3,21 @@ import { dasherize } from 'ember-runtime/system/string';
 
 function classHelper({ positional }) {
   let path = positional.at(0);
-  let propName = positional.at(1);
+  let truthyPropName = positional.at(1);
+  let falsyPropName = positional.at(2);
   let value = path.value();
 
   if (value === true) {
-    return dasherize(propName.value());
+    if (truthyPropName) {
```

Comment:
This case: https://github.com/emberjs/ember.js/pull/13337/files#diff-c47395a5aaf92aa0a7edbfa812d4e0afR218